config/constants.ts

```diff
@@ -99,6 +99,7 @@ Resources required for BaseSpace TES stack
  */
 
 export const basespaceAccessTokenSecretName = '/manual/BaseSpaceAccessTokenSecret'; // pragma: allowlist secret
+export const dataPortalApiUrlSsmParameterName = '/data_portal/backend/api_domain_name';
```

Comment:
Sorry Alexis, if I made you confuse...
This has to be SampleSheet check API domain parameter pointer.

I just check in SSM and, it is avail at this coordinate:

```
/sscheck/lambda-api-domain
```

```
export AWS_PROFILE=umccr-dev-admin
aws ssm get-parameter --name '/sscheck/lambda-api-domain' --output json | jq '.Parameter.Value'
```

And. Probably, all down under changes (vars, etc) have to re-align naming to reflect it is `samplesheet-check` or `sscheck` accordingly. 

It is NOT Portal API domain URL or domain name.

Reply:
Ah my bad, will fix this later tonight

Reply:
Done